src/legacy/core_plugins/kbn_vislib_vis_types/public/heatmap.d.ts

```diff
@@ -17,14 +17,18 @@
  * under the License.
  */
 
-import { uiModules } from 'ui/modules';
-import lineInterpolationOptionTemplate from './line_interpolation_option.html';
-const module = uiModules.get('kibana');
+import { ColorSchemas } from 'ui/vislib/components/color/colormaps';
+import { RangeValues } from 'ui/vis/editors/default/controls/ranges';
+import { CommonVislibParams, ColorSchemaVislibParams, ValueAxis } from './types';
+import { Positions } from './utils/collections';
 
-module.directive('lineInterpolationOption', function () {
-  return {
-    restrict: 'E',
-    template: lineInterpolationOptionTemplate,
-    replace: true
-  };
-});
+export interface HeatmapVisParams extends CommonVislibParams, ColorSchemaVislibParams {
+  type: 'heatmap';
```

Comment:
Let's add a definition for `times` prop as well.

Reply:
ok, done